### PR TITLE
 [docs] Update on containerd pre requisites

### PR DESCRIPTION
**Pull request by selansen**
Update containerd pre requisites information. As WMCO installs and
manages containerd runtime, it is recommanded not to install by user.

Signed-off-by: selansen <esiva@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,7 +214,8 @@ Cluster autoscaling is supported for Windows instances.
 - Define and deploy a [MachineAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-machineautoscaler), referencing a Windows MachineSet.
 
 ### Container Runtime
-Windows instances brought up with WMCO are set up with the containerd container runtime.
+Windows instances brought up with WMCO are set up with the containerd container runtime. As WMCO installs and manages runtime,
+it is recommanded not to preinstall containerd in Machineset or BYOH Windows instances.
```
Comment (alinaryan) [inline README.md:218]: s/recommanded/recommended/

Comment (alinaryan) [reply README.md:218]: s/Machineset/MachineSet/

**Review by alinaryan — COMMENTED** [review]
spelling nits:

File: README.md [hunk 2]
```diff
@@ -214,7 +214,8 @@ Cluster autoscaling is supported for Windows instances.
 - Define and deploy a [MachineAutoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html#configuring-machineautoscaler), referencing a Windows MachineSet.
 
 ### Container Runtime
-Windows instances brought up with WMCO are set up with the containerd container runtime.
+Windows instances brought up with WMCO are set up with the containerd container runtime. As WMCO installs and manages runtime,
```
Comment (sebsoto) [inline README.md:217]: and manages runtime -> and manages the container runtime

**Review by sebsoto — CHANGES_REQUESTED** [review]

**Comment by selansen** [comment]
cc : @mburke5678

**Pull request merged**